### PR TITLE
Update start script

### PR DESCRIPTION
**Pull request by clarkent86**
Updating to using array for STARTCOMMAND due to issues that `su steam -c "${STARTCOMMAND}"` can have with escaped quotes and spaces, also this will allow for spaces in server names and passwords

## Context 

Allow users to use spaces in environment variables

## Choices

- Better handling of spaces in program arguments that should allow them, such as server name and passwords

## Test instructions

1. Run the server with this change and have spaces in the passwords or in the server name

## Checklist before requesting a review

* [x] I have performed a self-review of my code
* [x] I've added documentation about this change to the README. 
there is no mention spaces aren't supported, so this will just be helpful overall
* [x] I've not introduced breaking changes.
fully tested this time instead of just testing printing the startcmd. sorry about that! here's proof:

image

### CODE REVIEW

File: scripts/start.sh [hunk 1]
```diff
@@ -75,5 +75,5 @@ password: ${ADMIN_PASSWORD}
 EOL
 
 printf "\e[0;32m*****STARTING SERVER*****\e[0m\n"
-echo "${STARTCOMMAND}"
-su steam -c "${STARTCOMMAND}"
```
Comment (clarkent86) [inline scripts/start.sh:79]: This ensures that the entire command string is correctly interpreted as a single argument to bash.

**Comment by clarkent86** [comment]
@thijsvanloef fixed and fully tested this time!

**Comment by clarkent86** [comment]
@Dashboy1998 looks like you were right about the `[*]` syntax vs. `[@]`. It looks to be honoring the spaces just fine, and the linter likes it better. Thanks again!

**Comment by clarkent86** [comment]
image

**Comment by clarkent86** [comment]
@thijsvanloef all set! Apologies for the delay. The only reason I've been able to focus on this and why I've been doing it in the middle of the night...I just had a baby girl! Our first one. Sorry to get personal, but I'm just too excited :D

**Comment by thijsvanloef** [comment]
@clarkent86 Wow! Congrats on the little girl! I wish you folks all the best. Ofcourse the baby comes first, don't worry about it!
I don't have time to review this today, will definitely do so tomorrow!

**Pull request merged**